src/test/resources/com/puppycrawl/tools/checkstyle/javadocpropertiesgenerator/InputJavadocPropertiesGeneratorCorrect2.java

```diff
@@ -0,0 +1,16 @@
+package com.puppycrawl.tools.checkstyle.javadocpropertiesgenerator;
+
+public class InputJavadocPropertiesGeneratorCorrect2 {
+
+    // note: &#064; is the html escape for '@',
+    // used here to avoid confusing the javadoc tool
+    /**
+     * An annotation of a package, type, field, parameter or variable.
+     * An annotation may occur anywhere modifiers occur (it is a
+     * type of modifier) and may also occur prior to a package definition.
```

Comment:
Can we remove text that does not matter.
If we care about first line, let's keep only it.

Reply:
done